_questions/machine-learning-zoomcamp/module-1-homework/008_y97mg42O7d_homework-q7-what-do-the-weights-represent.md

```diff
@@ -0,0 +1,11 @@
+---
+id: y97mg42O7d
```

Comment:
I wasn't sure what to use as a UUID so I just made one up using a simple Python function. I can change that if there is a preferred convention to follow.